PCIe 6.0 Packet Generator and Exerciser/PCIe 6.0 Packet Generator and Exerciser/Endpoint/configuration_algorithm_classes/completion_construction_algorithms.cpp

```diff
@@ -0,0 +1,40 @@
+#include "completion_construction_algorithms.h"
+
+CompletionConstructionAlgorithms::CompletionConstructionAlgorithms(): dataToBeReadBits(0){}
+
+/**
+ * @brief Constructing a Completion TLP with Data Payload inside (return of a Configuration Read request)
+ * 
+ * @return TLP -> The Constructed TLP (CplD)
+ */
+TLP CompletionWithData::constructTLP()
+{
+    TLP a;
+
+    return a;
+}
+
+/**
+ * @brief Constructing a Completion TLP without Data Payload inside (return of a Successful Configuration Write request)
+ * 
+ * @return TLP -> The Constructed TLP (Cpl)
+ */
+TLP CompletionWithoutData::constructTLP()
+{
+    TLP a;
+
+    return a;
+}
+
+/**
+ * @brief Constructing a Completion TLP with Unsupported request (OHC-A5) (return of an Unsuccessful Read or Write request)
+ * 
+ * @return TLP -> The Constructed TLP (CplUR)
+ */
+TLP CompletionWithUR::constructTLP()
+{
```

Comment:
This one as well should be implemented before merging

Reply:
same as above

Reply:
okay 👍